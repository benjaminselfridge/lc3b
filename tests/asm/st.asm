        0x5000

        LEA     R0      ADDR
        LDW     R0      R0      0x0
        ADD     R1      R1      0x003
        STW     R1      R0      0x0
        LDW     R2      R0      0x0
        TRAP    0x0

ADDR:   0x5100
