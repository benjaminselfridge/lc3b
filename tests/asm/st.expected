[ RegContains 0 0x5100
, RegContains 1 0x003
, RegContains 2 0x003
]